### DISCUSSION TITLE
waline评论加载成功后有没有回调函数？

### DISCUSSION THREAD

**Discussion by appotry** [walinejs/waline#1624] [Q&A] [answered]
加载评论成功后会导致waline区域高度发生较大变化，想做一些其它css修改优化，需要一个回调函数

另一个问题，默认可否给waline区域一个高度，在waline完全加载成功之前，占用一个给定参数高度，显示加载waline画面，例如一个动态loading 图片。waline加载完成后替换加载动画为评论区

**Comment by Mister-Hope** [comment, marked as answer]
我不能理解你在说什么。

1. Waline 提供的 init 函数是同步的
2. 没有限制说 waline 必须挂载到一个空元素上。
